### ISSUE TITLE
Admin Shop gives permission error

### ISSUE THREAD

**Issue by sjaustad** [ChestShop-authors/ChestShop-3#249]
Minecraft Version: **12.2.2**
ChestShop Version: **3.9.2**
iConomy VersionL **5.08 (Towny version)**

Server Setup: I am running a Cat Server so I can use both Forge and Bukkit plugins on the same server. I am also using Towny Advanced (v 93)

Here is my sign configuration
```
AdminShop (or "Admin Shop")
1
B 1:S 1 (or "B 1: 1 S")
?
```

**Expected Behavior:**
I right click on the sign with the item I want the shop to sell (e.g. bed) and the shop now sells bed to players.

**Actual behavior:**
I right click on the sign with the bed and I get the error: "You don't have permission to do that."

**Additional information:** 
I am Opped on the server. Normal player chest shops work just fine. If you have a chest nearby you can buy and sell items. Only the admin shop doesn't work. If I manually type in "Bed" instead of ? the shop just doesn't work at all.

I am using LuckPerm and ForgeEssentials but in a test environment I wiped away both of these permission mods and made sure I was opped and received the same error, so I think it's a permission issue with just the plugin. I also tried manually adding the permission chestshop.admin to my player and that did not work as well.

Any help would be appreciated.

**Comment by Phoenix616** [comment]
Please try with the latest 1.12 development build as well as with normal
user permissions (no special ones/op) and with actual CraftBukkit/Spigot if
the issue persists after that.

Also you might want to provide your missing config and log files as
requested in the issue template.

**Comment by sjaustad** [comment]
I tried the latest development build from [here](https://ci.minebench.de/job/ChestShop-3-1.12/). It did resolve the permissions issue but not every time someone tries to buy or sell from any shop, there is a "player not found" error.

**Comment by Phoenix616** [comment]
Hm, can you test it it works after a player shop is re-created?

**Comment by sjaustad** [comment]
Yeah that issue is from any shop whether it's admin or player. I also updated to latest vault version but that didn't fix anything. After this testing I installed MIG, which is another shop mod and it worked it just fine both buying and selling so I'm not sure what's up.

**Comment by Phoenix616** [comment]
Did the player re-create the player shop themselves or did you do that for them? (Make sure it was the player themselves without writing the name on it so that we can 100% sure that the auto-generated name doesn't work)

Also please provide your log and config file.
